translate/storage/catkeys.py

```diff
@@ -128,6 +132,13 @@ def settargetlanguage(self, newlang):
         self._header_dict['language'] = data.languages[newlang][0].lower()
     targetlanguage = property(None, settargetlanguage)
 
+    def setchecksum(self, checksum):
+        """Set the checksum for the file"""
+        if not checksum:
+            return
+        self._header_dict['checksum'] = str(checksum)
+    checksum = property(None, setchecksum)
```

Comment:
I don't see the need for this to be a property since we can just live with the `set_checksum` method.

Reply:
Done (to be consistent with underscore usage, I have named it setchecksum)